pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -251,21 +272,26 @@ def connection_select(context, name):
                      default='all'))
 @add_options(verify_option)
 @click.pass_obj
-def connection_add(context, **options):
+def connection_add(context, name, **options):
     """
     Add a new WBEM connection definition from specified options.
 
     Create a new WBEM connection definition in the connections file from
     the specified options. A connection definition with that name must not
     yet exist.
 
-    The current connection remains unchanged by this command.
+    The NAME argument MUST exist. It define the server uri
```

Comment:
Multiple issues in: "It define the server uri and the unique name under which this server connection information will be stored.":
* Grammar: "it define"
* It does not define the server URI.
If we follow my suggestion on changing the sentence on line 693, the whole sentence can be removed.

Reply:
DROPPED. DONE